.flake8

```diff
@@ -13,4 +13,4 @@
 [flake8]
 max-line-length = 80
 select = C,E,F,W,B,B950
-ignore = E501, W503
+ignore = E501, W503, E203
```

Comment:
I don't think we _always_ want to ignore this rule, though, right?

Reply:
I think we do; that is what is suggested by the Black documentation.

Reply:
https://github.com/psf/black/issues/315#issuecomment-395457972